classes/controller/FrontController.php

```diff
@@ -170,6 +170,14 @@ class FrontControllerCore extends Controller
      */
     protected $cccReducer;
 
+    /**
+     * Set this parameter to false if you don't want cart's invoice address
+     * to be set automatically (this bad behavior is kept for legacy and BC purpose)
```

Comment:
```suggestion
     * to be set automatically (this behavior is kept for legacy and BC purpose)
```
We are not sure this is bad because we dont have the history 😅

Reply:
well, I assume any code updating a specific field in the DB in `FrontController::init()` is bad 🤔 

But you're right, it's not a great idea to describe it like this in a comment, I'll update it right now 👍